### DISCUSSION TITLE
Some stuff broken when transferring user data files between different osu lazer installs on different oses

### DISCUSSION THREAD

**Discussion by bytenekos** [ppy/osu#25261] [Q&A] [answered]
### Type

Other

### Bug description

I tried copying over my user data files from my windows install into my macos install (hoping that all my replays and stuff would copy over from unranked maps) but as a result some maps appear to be broken once copied over, some replays also appear to just be broken (cursor doesn't move resulting in the entire map being missed).

I don't have osu stable installed, so the maps (in theory) should all just be there (I would have assumed)

### Screenshots or videos

https://youtu.be/iCUEducO8wU

### Version

2023.1026.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/13184845/database.log)
[input.log](https://github.com/ppy/osu/files/13184846/input.log)
[network.log](https://github.com/ppy/osu/files/13184847/network.log)
[performance.log](https://github.com/ppy/osu/files/13184848/performance.log)
[runtime.log](https://github.com/ppy/osu/files/13184849/runtime.log)

**Comment by peppy** [comment]
This sounds like you didn't copy all files. Maybe the file transfer was interrupted halfway through.

Please try copying again.

**Comment by bytenekos** [comment]
Tried copying again, exact same result. Same maps missing, same replays appearing (and not appearing)

**Reply by Joehuu** [reply, marked as answer]
You have to copy the whole `osu` folder, not just `files` (based on video provided).

**Comment by bytenekos** [comment]
> You have to copy the whole `osu` folder, not just `files` (based on video provided).

This works, but seems to have broken global hotkeys on mac (probably due to different oses having different ways for doing hotkeys maybe? idk)

**Reply by bytenekos** [reply]
Nevermind it's because I was in full screen
